mantidproject/mantid
# Adding a fitting range tool to MultiDatasetFitting interface

Trac ticket [#10474](http://trac.mantidproject.org/mantid/ticket/10474)

**To test**
- Start the interface from Interfaces->General->Multi dataset Fitting.
- Load some data
- Change fitting ranges for individual spectra and for all simultaneously.
- Fit some function. Check that the correct ranges are used.
